bin/run.py

```diff
@@ -158,6 +158,12 @@ def validate_format(
                     data = ret.err
                 elif ret.out:
                     data = ret.out
+
+                if validator_type == 'input_format':
+                    file = self.in_path
+                if validator_type == 'output_format':
+                    file = self.ans_path
+                data += f'{Style.RESET_ALL}in {shorten_path(self.problem, file.parent) / file.name}\n'
```

Comment:
I suppose you tested this and it looks decent in the terminal?

Reply:
yea thats why i shortened the paths, looks like this:
```
String length 1000001 out of range [2, 1000001)! Line: 1, Char: 0
((()()((((()((((()((()()(()()))()))))(()())())((((()(((())))((()))((()(()(([...]
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
in /tmp/bapctools_98c90c/loopinvariant/links/6eefaf/testcase.in
```